app/src/main/res/values/strings.xml

```diff
@@ -41,6 +42,8 @@ to learn more.
     <string name="choose_username_info">This is how others will see you on other apps.</string>
     <string name="choose_username">Choose Username</string>
     <string name="username_not_available">Username is not available</string>
+    <string name="continue_no_username"><u>Continue without username</u></string>
+    <string name="continue_no_username_info">Identities with no username may have problems with some applications …</string>
```

Comment:
The ... make it feel like some of the text was cut. A different message suggestion:
`Some applications might require an account with username.`
or
`Some applications might require username`
And maybe the confirm button should say something like "Continue" instead of "OK", which does not feel like a confirmation.